### ISSUE TITLE
I CAN'T GET IT TO WORK?

### ISSUE THREAD

**Issue by WertyMaster** [Jire/Overwatcheat#141]
OK FIRTS.. SORRY FOR MY BAD ENGLISH ...
SO I COMPILED THE BUILD ,EVERYTHING SEEMS OK 
I INSTALLED INTERCEPTION AND PUT THE .DLL FILE IN THE BUILD
BUT WHEN I START THE CHEAT NOTHING WORKS, IT CHANGES NOTHING 
I TRIED OBS , IT DIDN'T WORK ...
![Capture d’écran (56)](https://user-images.githubusercontent.com/115594830/196007118-834e61ce-cf15-468c-a8e0-490e78169b32.png)
![Capture d’écran (57)](https://user-images.githubusercontent.com/115594830/196007122-957e49b8-6c4d-45c6-a40e-8daa8a036f6b.png)
![Capture d’écran (58)](https://user-images.githubusercontent.com/115594830/196007124-04722d71-c760-4e66-9af8-9782ba7d0497.png)

**Comment by Ceozz** [comment]
i have the same issue idk it doesnt work for overwatch 2 for me

**Comment by obelisk9911** [comment]
Same issue

**Comment by rmgnum** [comment]
Are you playing in borderless window?

**Comment by lihuate** [comment]
The same problem occurs again, so how can we avoid it thoroughly？O(∩_∩)O
![image](https://user-images.githubusercontent.com/26449899/196357198-750b6d03-d91d-4c8e-8164-33cdf51ca2d1.png)

**Comment by WertyMaster** [comment]
> Are you playing in borderless window?

yes, i tried in borderless in full and i tried change value to desktop ..none work

**Comment by rmgnum** [comment]
> > Are you playing in borderless window?
> 
> yes, i tried in borderless in full and i tried change value to desktop ..none work

Because of your settings. It clearly says "# When using flicking mode, do not set this to left mouse button."

Change your aim_key to your preference or disable flick option

**Comment by Subtraxtt** [comment]
> i have the same issue idk it doesnt work for overwatch 2 for me

same issues pls someone help
